src/sentry/utils/email.py

```diff
@@ -35,7 +35,7 @@
 from sentry.utils.strings import is_valid_dot_atom
 from sentry.web.helpers import render_to_string
 
-logger = logging.getLogger(__name__)
+logger = get_logger()
```

Comment:
Does `__name__` get implied somehow with structlog? Or should we be doing `get_logger(name=__name__)`?